Doc/tools/extensions/pyspecific.py

```diff
@@ -58,11 +59,33 @@
 
 def issue_role(typ, rawtext, text, lineno, inliner, options={}, content=[]):
     issue = utils.unescape(text)
+    # sanity check: there are no bpo issues within these two values
+    if int(issue) > 47261 and int(issue) < 400000:
```

Comment:
Can be simplified a bit:

```suggestion
    if 47261 < int(issue) < 400000:
```